docs/news.rst

```diff
@@ -1,26 +1,73 @@
+.. raw:: html
 
+    <div class = "twitter-block"  style = "width:100%; display: flex; justify-content: center;">
+    <a class="twitter-timeline" data-width="850" data-height="900" href="https://twitter.com/tardis_sn?ref_src=twsrc%5Etfw">Tweets by tardis_sn</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+    </div>
 
-GSOC 2017
----------
 
-.. image:: https://developers.google.com/open-source/gsoc/resources/downloads/GSoC-logo-horizontal.svg
-  :target: https://summerofcode.withgoogle.com/organizations/4793296782098432/
-  :width: 458 px
 
-TARDIS participates in the Google Summer of Code 2017 program as part of the Python Software Foundation!
+    <script>
+        document.getElementsByClassName("twitter-block")[0].addEventListener(
+          "DOMSubtreeModified",
+          function (e) {
+            for (var target = e.target; target && target != this; target = target.parentNode) {
+              // loop parent nodes from the target to the delegation node
+              if (target.matches("#twitter-widget-0")) {
+                customizeTweetMedia();
+              }
+            }
+          },
+          false
+        );
 
+        var customizeTweetMedia = function () {
+          var iframe = document.getElementById("twitter-widget-0");
+          var headerTitle = iframe.contentWindow.document.getElementsByClassName("timeline-Header-title")[0];
+          headerTitle.style.fontSize = "18px";
 
-SOCIS 2016
-----------
+          var headerTitleByInline = iframe.contentWindow.document.getElementsByClassName("timeline-Header-byline")[0];
+          headerTitleByInline.style.fontSize = "18.5px";
 
-TARDIS is part of ESA's Summer of Code in Space 2016 program!
+          var tweetTextList = iframe.contentWindow.document.getElementsByClassName("timeline-Tweet-text");
+          var index;
+          for (let index = 0; index < tweetTextList.length; index++) {
+            tweetTextList[index].style.fontSize = "16px";
+            tweetTextList[index].style.font = "Lato,proxima-nova,Helvetica Neue,Arial,sans-serif;";
```

Comment:
Most probably it won't work - have you checked the font being used by web inspector on built webpage?

You should instead do this (read [here](https://www.w3schools.com/jsref/prop_style_fontfamily.asp)]:
```suggestion
            tweetTextList[index].style.fontFamily = "Lato, proxima-nova, Helvetica Neue, Arial, sans-serif";
```

Reply:
Oh, my bad, I fixed it now. Thanks again!
Could you please check [this link](https://atharva-2001.github.io/tardis/branch/news_feed_docs/) to view the documentation?